src/mongocrypt-kms-ctx.c

```diff
@@ -1448,9 +1639,18 @@ _mongocrypt_kms_ctx_init_kmip_register (mongocrypt_kms_ctx_t *kms_ctx,
    const uint8_t *reqdata;
    size_t reqlen;
 
+   if (!kms_ctx) {
```

Comment:
Suggest converting into a not-null assertion.

Reply:
Changed as suggested.  Will show up in next commit.